core/startos/src/service/service_effect_handler.rs

```diff
@@ -328,8 +328,23 @@ async fn get_container_ip(context: EffectContext, _: Empty) -> Result<Ipv4Addr,
 async fn get_service_port_forward(
     context: EffectContext,
     data: GetServicePortForwardParams,
-) -> Result<Value, Error> {
-    todo!()
+    host_id: HostId,
```

Comment:
this should be part of `GetServicePortForwardParams`

Reply:
done